packages/vue/package.json

```diff
@@ -4,13 +4,14 @@
   "description": "Carbon charting components for Vue",
   "main": "index.js",
   "files": [
-    "dist",
-    "index.js",
+    "charts-view.common.js",
+    "charts-view.umd.js",
+    "charts-view.umd.min.js",
```

Comment:
...-vue?

Reply:
ahahaha, yeah I actually meant `-vue`

Reply:
You would not believe the number of times I've don that

Reply:
😄